esrc/rescueoverview.php

```diff
@@ -382,22 +390,24 @@ function displayLine($row, $charname, $finished, $system, $notes, $isCoord, $sum
 				$row['pilot'] .'">'.Output::htmlEncodeString($row['pilot']).'</a></td>';
 	}
 	
-	// Status 
-	// set status color: green = new, yellow = pending, orange = check-in needed
-	switch ($status) {
-		case 'new':
-			$statuscellformat = ' style="background-color:green;color:white;"';
-		break;
-		case 'pending':
-			$statuscellformat = ' style="background-color:yellow;color:black;"';
-		break;
-		default:
-			$statuscellformat = '';
-		break;
-	}
-	if ($finished == 0 && strtotime($row['lastcontact']) < strtotime('-7 day')) {
+	// Status - set status color: 
+	// green = new, 
+	// yellow = pending
+	// orange = check-in needed,
+	// red = check killboard
+	
+	if ($status == 'check-killboard') {
+		$statuscellformat = ' style="background-color:red;color:white;"';
```

Comment:
Should we display a linke (somehow) to the pilots zkill stats if he is active? Just to speedup the process for the SAR coord. And should open the zkill in a new window to still stay on current page.